lib/index.js

```diff
@@ -13,7 +13,8 @@ let DEFAULTS = {
 	}
 };
 
-module.exports = (referenceDir, config, { watch, fingerprint, compact }) => {
+module.exports = (referenceDir, config,
+		{ watch, fingerprint, compact, "source-maps": sourceMaps }) => {
```

Comment:
As discussed elsewhere, IMO the CLI module should map `--source-maps` to `sourcemaps` - alternatively, we might just get rid of the hyphen altogether (i.e. `--sourcemaps`). /cc @moonglum